lib/AST/Module.cpp

```diff
@@ -1688,9 +1688,12 @@ LookupConformanceInModuleRequest::evaluate(
       if (nominal->isMoveOnly()) {
         return ProtocolConformanceRef::forInvalid();
       } else {
-        // FIXME: this should probably follow the Sendable case in that
-        // we should synthesize and append a ProtocolConformance to the `conformances` list.
-       return ProtocolConformanceRef(protocol);
+        // Specifically do not create a concrete conformance to Copyable. At
+        // this stage, we don't even want Copyable to appear in swiftinterface
+        // files, which will happen for a marker protocol that's registered
+        // in a nominal type's conformance table. We can reconsider this
+        // decision later once there's a clearer picture of noncopyable generics
```

Comment:
I think other parts of the code won't expect to see an abstract conformance when the conforming type is not a type parameter or archetype. Perhaps the swiftinterface printing code should be changed instead? I think it already knows to skip Sendable conformances in some cases IIRC.

Reply:
i will have to address that later. i'm not sure yet if we want concrete conformances at all. it's likely that Copyable will get revised into a LayoutConstraint.